src/test/java/org/commcare/formplayer/tests/EndpointLaunchTest.java

```diff
@@ -123,6 +129,21 @@ public void testEndpoints() throws Exception {
                         "f04bf0e8-2001-4885-a724-5497b34abe95"});
     }
 
+    @Test
+    @WithHqUser(enabledToggles = {TOGGLE_SESSION_ENDPOINTS})
+    public void testEndpointsWithInlineCaseSearch() throws Exception {
+        configureQueryMock();
+        HashMap<String, String> endpointArgs = new HashMap<>();
+        endpointArgs.put("case_id", "0156fa3e-093e-4136-b95c-01b13dae66c6");
+        CommandListResponseBean commandListResponse = sessionNavigateWithEndpoint(APP_NAME,
+                "inline_w_display_cond_case_list",
+                endpointArgs,
+                CommandListResponseBean.class);
+        assert commandListResponse.getCommands().length == 2;
```

Comment:
Good call. A case is needed so it can be selected and move onto the next menu. Of the two forms, I made one display condition false and the other true. Now the test will auto select the visible one and auto advance into the visible form.

[b92cbdc](https://github.com/dimagi/formplayer/pull/1608/commits/b92cbdc583df07c96d3d16832f07f93471390657)